### ISSUE TITLE
Do we need bulk selection mechanism and support for bulk operations?

### ISSUE THREAD

**Issue by krukhlis** [mbrlabs/Mundus#33]
Hi All, 
Want to use this issue as disc-n for the titled topic.
IMHO, bulk operations are really important for any world editor:
1) we can apply the same material to multiple game objects
2) we can set the same orientations or scale to the group of objects or translate them by some coordinates difference.
3) in real engine we will also specify game properties, kinda speed, stamina, health, etc.
4) etc.

The multiple selection should be possible from the tree navigator and with ribbon lasso in the world's view, e.g.:
![image](https://cloud.githubusercontent.com/assets/18131621/19279589/05112420-8fb0-11e6-8466-856291bac428.png)

What do you think guys?

**Comment by codenigma** [comment]
I think it would be a good feature to boost productivity. But at the moment there is still
missing a lot more important stuff ( the issues list should be updated in near future ). I think this feature depends heavily on the current code base, which could change when we start implementing the libgdx runtime. Maybe we should collect these feature wishes in a backlog for the moment.

**Comment by mbrlabs** [comment]
I agree with both of you. It' definitely necessary, but right now my main focus is still on the asset management (internally as well the ui). I was also thinking about implementing a generic material system, that you can apply to renderable components. I really think we should get this done right before moving on to other things, because that's gonna be a pretty important part for the editor and the runtime.

Sure, we can select some ideas [here](https://github.com/mbrlabs/Mundus/projects/2)

**Comment by krukhlis** [comment]
@mbrlabs , just wondering if you have somewhere document or backlog of the actual features that are required for Mundus? Some roadmap with priorities. I think such asset will be really helpful -- we can see what is urgent and help with these features as TOP-Priority.
Because right now -- there is Mundus with functionality to add models, generate terrain and put models to terrain. And it's unclear what are the next steps, vision and intentions. :)

**Comment by mbrlabs** [comment]
I need a couple of days to finish the base architecture of the asset system. I'm afraid there's not much to for you guys, because i'm still experimenting.
The focus for v0.1.0 will be the libgdx runtime + other editor features such as multi select.
If you guys have ideas, just open some issues and we can bundle them for the next milestone. 
I will also write a roadmap once we're there ;)
